contrib/ci-horizen/scripts/test/clang-tidy-launcher.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Get the current commit hash
+export COMMIT=`git log -1 --format="%H"`
+# Get the names of the files touched by this commit
+export FILELIST=`git diff-tree --no-commit-id --name-only -r $COMMIT`
+
+# Filter out all files which do not end with .cpp, .h and .cc
+FILTEREDFILELIST=()
+for i in $FILELIST;
+do
+    if [[ $i =~ .*\.cpp$ ]] || [[ $i =~ .*\.h$ ]] || [[ $i =~ .*\.tcc$ ]];
```

Comment:
Is `[[ $i =~ .*\.tcc$ ]]` a typo for [[ $i =~ .*\.cc$ ]] (based on above comment)? Or do we want to include both `.cc` and `.tcc` files (in this case both the comment and the filter need to be updated)?

Reply:
I actually wanted to include only the `*.tcc` files, as all the `*.cc` included in the repository belong to leveldb. Instead, I would add the filter for `*.hpp` files, as you stated in another comment.